react/typings/vtex.checkout-resources.d.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-duplicates, import/order */
+
+declare module 'vtex.checkout-resources/QueryProfile' {
+  import { DocumentNode } from 'graphql'
+
+  const query: DocumentNode
+  export default query
+}
```

Comment:
Could you test something for me? Try changing the extensions of the `checkout-resources` queries/mutations from `.js` to `.ts(x)` and see if their types are exported, so you won't need this file anymore.

Reply:
I'll try!

Reply:
done